pkg/servicescm/servicescm.go

```diff
@@ -99,6 +102,21 @@ func NewData(services *[]Service, files *[]FileInfo) (*Data, error) {
 	return cmData, nil
 }
 
+// ListServiceConfigMaps returns a list of all windows-services ConfigMaps in the given namespace
+func ListServiceConfigMaps(c client.Client, ctx context.Context, namespace string) ([]core.ConfigMap, error) {
```

Comment:
This can just be `List` as the call site will read `servicevm.List()` and it being a list implies plural.